src/utils/utils.cpp

```diff
@@ -243,7 +243,7 @@ void mp::utils::wait_until_ssh_up(VirtualMachine* virtual_machine, std::chrono::
                                   std::function<void()> const& ensure_vm_is_running)
 {
     mpl::log(mpl::Level::debug, virtual_machine->vm_name,
-             fmt::format("Trying SSH on {}:{}", virtual_machine->ssh_hostname(), virtual_machine->ssh_port()));
+             fmt::format("Trying SSH on {}:{}", virtual_machine->ssh_hostname(timeout), virtual_machine->ssh_port()));
```

Comment:
Shouldn't we just _not_ call `::ssh_hostname()` here? Like… we have the `try_action_for()` below that's meant to handle timeouts. Worst case we'll wait 2*timeout here.

Reply:
What would we call instead?  This for *only* getting the hostname/IP address and not for seeing when SSH is up and running.

Reply:
Or ditch the log message or at least remove the IP/port info?

Reply:
Also, if not here, then the timeout will have to be added to
https://github.com/canonical/multipass/blob/d8d8b2149d8b72a1d9509929e0527d21ac7eddd8/src/utils/utils.cpp#L251
instead.

So, let me know how you prefer this to be handled.

Reply:
> Or ditch the log message or at least remove the IP/port info?

Yeah, we shouldn't block in this just to log it. Can log after we get the info in the action.

> Also, if not here, then the timeout will have to be added to

Will it? Isn't `try_action_for` dealing with that exactly?

Reply:
Actually, having discovered the LXD instance creation timeout and fixing that by moving it into `ensure_vm_is_running()`, I think this message should not include the hostname/IP address & port info.

Reply:
Ok, I modified this message to not get the hostname/port and just log a generic message.  I also modified the subsequent `ssh_hostname()` call to have a 1ms timeout so it will use this action instead of the one built into `ssh_hostname()`.